core/templates/pages/donate-page/donate-page.component.html

```diff
@@ -141,7 +141,23 @@ <h3 class="oppia-donate-h3">
     font-size: 0.825em;
   }
 
-  @media only screen and (max-width: 1160px) {
+  @media (max-width: 480px) {
```

Comment:
I would do this even for more width. Like 600 or 800, maybe upload videos so that I can see how it looks.

Reply:
I made the changes you suggested for 600px and 800px. Can you please check which one is better so that I can commit that one. @vojtechjelinek PTAL. Thanks!
**600px:**

[Screencast from 16-07-22 09:07:57 AM IST.webm](https://user-images.githubusercontent.com/86031203/179337910-68bd51df-a374-45c4-9926-8c4079eff61a.webm)

**800px:**

[Screencast from 16-07-22 09:09:47 AM IST.webm](https://user-images.githubusercontent.com/86031203/179337953-ec3326aa-757e-4b21-b8ca-ddd413648d61.webm)

Reply:
600px seems better